lib/fs.dart

```diff
@@ -10,7 +10,7 @@ export 'fs_status_listener.dart';
 /// Also note that methods may drop data if called when there is no Fullstory session active.
 class FS {
   /// Version of the underlying native Fullstory framework, e.g. '1.54.0'
-  static Future<String?> fsVersion() {
+  static Future<String?> get fsVersion {
```

Comment:
Thoughts on changing this to just 'version' to reduce redundancy a tiny bit?

Reply:
Yeah, we originally called it `version` in Android and iOS, but that conflicted with a built-in `version` property in iOS, so we renamed it to `fsVersion` across the board: https://github.com/cowpaths/mn/pull/81857#issuecomment-2437876370

In this case, I also think it helps clarify that this is the version of the underlying Android/iOS SDK, not the version of the `fullstory_flutter` plugin.

Reply:
Also, an async getter feels a little weird to me, but I'll trust your gut on this one.